frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -20,9 +21,12 @@ const normalizePackageManifests = (packageManifests, kind) => {
     const iconObj = _.get(packageManifest, 'status.channels[0].currentCSVDesc.icon[0]');
     const tileImgUrl = iconObj && `data:${iconObj.mediatype};base64,${iconObj.base64data}`;
     const tileIconClass = tileImgUrl ? null : iconClass;
-    const tileDescription = packageManifest.metadata.description;
     const tileProvider = _.get(packageManifest, 'metadata.labels.provider');
     const tags = packageManifest.metadata.tags;
+    const version = _.get(packageManifest, 'status.channels[0].currentCSVDesc.version');
+    const currentCSVAnnotations = _.get(packageManifest, 'status.channels[0].currentCSVDesc.annotations', {});
+    const { description, certifiedLevel, healthIndex, repository, containerImage, createdAt, support } = currentCSVAnnotations;
+    const tileDescription = description;
```

Comment:
I'd suggest using `description` rather than `tileDescription` throughout. Probably same for `tileProvider` -> `provider`.